api/src/main/java/com/datastrato/gravitino/authorization/SecurableObjects.java

```diff
@@ -144,16 +193,19 @@ private static void checkCatalog(SecurableObject catalog) {
     }
   }
 
-  private static final SecurableObject ALL_CATALOGS = new SecurableObjectImpl(null, "*");
+  private static final SecurableObject ALL_METALAKES =
+      new SecurableObjectImpl(null, "*", SecurableObjectType.CATALOG);
```

Comment:
Is this type correct?

Reply:
My fault. I will add some tests for this class.